### ISSUE TITLE
Collections in list are not reached from tabbing

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4192]
**Describe the bug**
The collections are not reached when tabbing in the modal.

**Where was the bug observed**
Local dev in branch `modals_acc` right now, pr: https://github.com/ushahidi/platform-client/pull/1637

**To Reproduce**
1. Go to collections through tabbing
2. Continue tabbing in the modal
3. The list of collections is not reached

**Expected behavior**
List of collections should be reached

**Is there a workaround? What is it.**
Not via tabbing, but it works when clicking

**Screenshots**
<img width="541" alt="Screenshot 2021-01-21 at 19 05 05" src="https://user-images.githubusercontent.com/8624777/105392604-aa8a6100-5c1b-11eb-99f3-930f591435cc.png">
